### ISSUE TITLE
 File "<unknown>", line 2     <!DOCTYPE html>     ^ SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by kkraoj** [samapriya/geeup#42]
Thank you for creating and maintaining this awesome repo. I am facing an auth issue lately. 
`geeup cookie_setup` works fine. It finishes with a "Cookie setup complete" result and also clears the terminal (like it used to). 

But just a few seconds later, when I try `geeup upload` with my files and meta_data csv, it says, cookies expired, please enter new cookies. I tried pasting the same cookies (as used for the `geeup cookie_setup`) and separately, also tried pasting new cookies (fetched from a new incognito window), but I get the same error. The error message is below.

```
Traceback (most recent call last):
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\kkrao\AppData\Local\conda\conda\envs\py36\Scripts\geeup.exe\__main__.py", line 7, in <module>
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\site-packages\geeup\geeup.py", line 653, in main
    func(args)
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\site-packages\geeup\geeup.py", line 350, in upload_from_parser
    pyramiding=args.pyramids,
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\site-packages\geeup\batch_uploader.py", line 101, in upload
    google_session = __get_google_auth_session(user, password, method)
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\site-packages\geeup\batch_uploader.py", line 380, in __get_google_auth_session
    and ast.literal_eval(response.text)["url"] is not None
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\ast.py", line 48, in literal_eval
    node_or_string = parse(node_or_string, mode='eval')
  File "c:\users\kkrao\appdata\local\conda\conda\envs\py36\lib\ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 2
    <!DOCTYPE html>
    ^
SyntaxError: invalid syntax
```
It seems like a google auth issue. Any workarounds or hints available for how to debug this?
`geeup` has been working fine for me all this while, so this seems like an issue caused by something external to `geeup`. I am running the latest version of `geeup` = 0.5.2.

**Comment by samapriya** [comment]
Can you try copying the cookies from a regular browser window, I have started to notice if you copy the cookies from an incognito browser window some pieces won't load or get copied. It seems to be working for me, but I have also seen some intermittent issues once in a while. Could you also past the overall command you are issuing for your workflow just to be sure there are no issues there

**Comment by kkraoj** [comment]
oh wow. copying cookies from a non-incognito window worked!

I will use cookies from non-incognito from now on. I noticed that the length of the cookie list is ~6k characters from non-incognito window and ~5k characters from the incognito window, even though in both cases, the cookie list is complete (both end with `{]`. So seems like some keys are missing in cookie list from incognito window. I am unable to tell which keys because once I generated the cookie list from normal window, even the incognito window is returning a cookie list of the same ~6k character size. I am using the "Copy Cookies" Chrome plugin to copy the cookies. 

For completeness, I am including the command I used for upload: 
`geeup upload --source "D:\Krishna\projects\vwc_from_radar\data\map\dynamic_maps\lfmc" --dest "users/kkraoj/lfm-mapper/lfmc_col_25_may_2021" -m "D:\Krishna\projects\vwc_from_radar\gee-app\upload_meta_data.csv" --nodata -9999 -u kkrao.j@gmail.com --method cookies		
`
